### DISCUSSION TITLE
Installation with another user

### DISCUSSION THREAD

**Discussion by Deathon** [pimcore/pimcore#17769] [Q&A / Support]
Hi guys, 

Have you tried installing pimcore with another user other than root?

**Comment by AlephLau** [comment]
What do you mean? Do you want to change the owner of the project files? @Deathon

**Comment by Deathon** [comment]
No i want install pimcore with user pimcore. I need to install pimcore with user pimcore and not with root user. Do you have any advice ?

**Reply by AlephLau** [reply]
@Deathon Sorry for the delay, here you can take a look at this, and it will show you how to create new users: https://pimcore.com/docs/platform/Pimcore/Administration_of_Pimcore/Users_and_Roles/
